README.md

```diff
@@ -1,34 +1,21 @@
 # GslibIO.jl
 
-Utilities to read/write *extended* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
+Utilities to read/write *extended* and *legacy* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
```

Comment:
Users may confuse that the link provided includes the extended format, but only includes the legacy one.

Reply:
That is a good point, but we don't have a link to the extended one. So what can we do? Any suggestion to rephrase the sentence? I think it should be fine as most users will already be familiar with GSLIB anyways, so they will know what this is about.

Reply:
I understand that extended format is the proposed format of this package. I am not aware of any formal extended GSLIB format (except this one). In that sense I would suggest something like this:
```
Extend the GSLIB format [GSLIB](http://www.gslib.com/gslib_help/format.html) and provide utilities to read/write files in Julia. For backward compatibility it also provides utilities to read/write *legacy* format.
```

Reply:
I think the proposed sentence is more confusing than the current one. The current one is to the point: it says what the package provides right away. I understand we extend the format, but that is not the main contribution of the software. Let's keep the current sentence as is and review it in the future if needed. It should be clear enough. 👍🏽